simulation/traffic_simulator/include/traffic_simulator/math/transfrom.hpp

```diff
@@ -23,6 +23,10 @@ namespace math
 {
 const geometry_msgs::msg::Pose getRelativePose(
   const geometry_msgs::msg::Pose & from, const geometry_msgs::msg::Pose & to);
+const geometry_msgs::msg::Point transformPoint(
+  const geometry_msgs::msg::Pose & pose, const geometry_msgs::msg::Point & point);
+std::vector<geometry_msgs::msg::Point> transformPoints(
+  geometry_msgs::msg::Pose pose, std::vector<geometry_msgs::msg::Point> points);
```

Comment:
```
std::vector<geometry_msgs::msg::Point> transformPoints(
  geometry_msgs::msg::Pose pose, std::vector<geometry_msgs::msg::Point> points);
```
↓
```
std::vector<geometry_msgs::msg::Point> transformPoints(
  const geometry_msgs::msg::Pose & pose, const std::vector<geometry_msgs::msg::Point> & points);
```

Reply:
Fixed in https://github.com/tier4/scenario_simulator_v2/pull/633/commits/8a484b8ff81788c5188b6d779e81b84a87eff5be